contributors/devel/architectural-roadmap.md

```diff
@@ -275,13 +275,13 @@ Kubernetes cannot function without this basic API machinery and semantics, inclu
   /portforward APIs.
 
 * TBD: The
-  [CertificateSigningRequest](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/kubelet-tls-bootstrap.md)
+  [CertificateSigningRequest](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/kubelet-tls-bootstrap.md)
```

Comment:
same repo. This can be:
`[CertificateSigningRequest](/contributors/design-proposals/cluster-lifecycle/kubelet-tls-bootstrap.md)`